addons/web/controllers/main.py

```diff
@@ -424,6 +424,20 @@ def xml2json_from_elementtree(el, preserve_whitespaces=False):
     res["children"] = kids
     return res
 
+def _password_warn(env, partner):
+    """ Admin still has `admin` password, flash a message via chatter
+
+    Does not use message_post because depending on conf' that might send an
+    email instead.
+    """
+    MailChannel = env['mail.channel']
+    MailChannel.browse(MailChannel.channel_get([partner.id])['id'])\
+        .message_post(
+            body=_("Your administrator password is the default (admin), if this is any sort of production database you may want to change that for security reasons."),
```

Comment:
```suggestion
            body=_("Your password is the default (admin)! If this system is exposed to untrusted users it is important to change it immediately for security reasons. I will keep nagging you about it!"),
```

Rationale: it is important to do it _also_ for test/dev databases when they are exposed publicly. They can be abused too in various ways.